src/main/kotlin/racingcar/domain/CarFactory.kt

```diff
@@ -0,0 +1,5 @@
+package racingcar.domain
+
+object CarFactory {
+    fun generateCars(carNames: List<String>) = List(carNames.size) { Car(carNames[it]) }
```

Comment:
작성하신 방식이 잘못된 것은 아니지만 
이 부분은 아래와 같이 변경하는게 조금 더 깔끔하지 않을까싶어요. 😃
```suggestion
    fun generateCars(carNames: List<String>) = carNames.map { Car(it) }
```